src/dtx/dtx_common.c

```diff
@@ -482,43 +482,49 @@ dtx_aggregation_pool(struct dss_module_info *dmi, struct dtx_batched_pool_args *
 				    dbca_pool_link);
 		D_ASSERT(!dbca->dbca_deregister);
 
+		dtx_get_dbca(dbca);
```

Comment:
I'm wondering how can this solve the issue I mentioned in prior comment? Looks dtx_free_dbca() only checks the dbca refcount at the end.

Reply:
> I'm wondering how can this solve the issue I mentioned in prior comment? Looks dtx_free_dbca() only checks the dbca refcount at the end.

dtx_free_dbca() call sched_req_put() will when xxx_done is true, at that time, the sched_req_put() will not yield, so only one will call sched_req_put().